cmd/fleet/cron.go

```diff
@@ -164,14 +166,31 @@ func scanVulnerabilities(
 
 	nvdVulns := checkNVDVulnerabilities(ctx, ds, logger, vulnPath, config, vulnAutomationEnabled != "")
 	ovalVulns := checkOvalVulnerabilities(ctx, ds, logger, vulnPath, config, vulnAutomationEnabled != "")
-	vulns, meta := recentVulns(ctx, ds, logger, nvdVulns, ovalVulns, config.RecentVulnerabilityMaxAge)
+	checkWinVulnerabilities(ctx, ds, logger, vulnPath, config, vulnAutomationEnabled != "")
 
-	if len(vulns) > 0 {
+	// If no automations enabled, then there is nothing else to do...
+	if vulnAutomationEnabled == "" {
+		return nil
+	}
+
+	vulns := make([]fleet.SoftwareVulnerability, 0, len(nvdVulns)+len(ovalVulns))
+	vulns = append(vulns, nvdVulns...)
+	vulns = append(vulns, ovalVulns...)
+
+	meta, err := ds.ListCVEs(ctx, config.RecentVulnerabilityMaxAge)
+	if err != nil {
+		errHandler(ctx, logger, "could not fetch CVE meta", err)
+		return nil
+	}
+
+	recentV, matchingMeta := utils.RecentVulns(vulns, meta)
+
+	if len(recentV) > 0 {
 		switch vulnAutomationEnabled {
 		case "webhook":
 			args := webhooks.VulnArgs{
```

Comment:
So... with this we would be triggering webhooks/jira/etc with Windows OS vulnerabilities but without any UI. Is this understanding correct?

Reply:
No, the value for `recentV` is only based on `nvdVulns` and `ovalVulns`: https://github.com/fleetdm/fleet/pull/7889/files/be782113bba5f33ae1c5844d0f6c4a03bd03432f#diff-fba00e967cbf619e2975f20568a79e6d06d61cb6b5adfba031c3d98f544d4c19R176